Forms/FormField.php

```diff
@@ -1501,27 +1501,23 @@ public function getSchemaState() {
 	 * @return array
 	 */
 	public function getSchemaStateDefaults() {
-		$field = $this;
-		$form = $this->getForm();
-		$validator = $form ? $form->getValidator() : null;
-		$errors = $validator ? (array)$validator->getErrors() : [];
-		$messages = array_filter(array_map(function($error) use ($field) {
-			if($error['fieldName'] === $field->getName()) {
-				return [
-					'value' => $error['message'],
-					'type' => $error['messageType']
-				];
-			}
-			return null;
-		}, $errors));
-
-		return [
+		$state = [
+            'name' => $this->getName(),
 			'id' => $this->ID(),
 			'value' => $this->Value(),
-			'valid' => (count($messages) === 0),
-			'messages' => (array)$messages,
+			'message' => null,
 			'data' => [],
 		];
+		
+		if ($message = $this->Message()) {
+			// @todo - Make form / field messages not always stored as html
```

Comment:
Inline todos should use `TODO` - no biggie, just sayin. I think this is more commonly picked up by IDEs and other code cleanliness tools. A single comment doesn't start a PHPDoc block, and `@todo` is PHPDoc notation